tst/regression/test_suites/aniso_therm_cond_gauss_conv/aniso_therm_cond_gauss_conv.py

```diff
@@ -137,10 +141,15 @@ def get_ref(x, Bx, field_cfg):
             outname = get_outname(all_cfgs[step])
             data_filename = f"{parameters.output_path}/parthenon.{outname}.final.phdf"
             data_file = phdf.phdf(data_filename)
-            prim = data_file.Get("prim")
             zz, yy, xx = data_file.GetVolumeLocations()
             mask = yy == yy[0]
-            temp = prim[4][mask]
+            # Flatten=true (default) is currently (Sep 24) broken so we manually flatten
```

Comment:
Is this an h5py issue, or something on the Parthenon side?

Reply:
Parthenon. `Get()` by default has `flatten=True` but instead of a flat array an array with a leading block index is returned. Needs to be fixed upstream, but otherwise the name components stuff is nicer than the hardcoded old indices.